giza/giza/content/options/tasks.py

```diff
@@ -44,8 +44,16 @@ def option_tasks(conf):
 
     tasks = []
     for dep_fn, option in o.content_iter():
+        if option.has_field('command'):
```

Comment:
This is admittedly janky. The options files for the mongoreplay commands use `program: mongoreplay commandname` as their "program", so this chops that up. It's probably not handling errors well, but it's also kind of crazy so I figured you might have thoughts.